data/ships/ac33.json

```diff
@@ -1,6 +1,7 @@
 {
 	"model": "ac33",
 	"name": "AC-33 Dropstar",
+	"i18n_key": "AC33",
```

Comment:
* @sturnclaw Is this what you had in mind?
* What do I do with the missiles. Should they also have the i18n_key tags?

Reply:
> @sturnclaw Is this what you had in mind?

Looks like I need a common component like SHIP.

```json
"i18n_key": "SHIP_AC33",
```

Reply:
Fixed. I've looked at how it's done with the hardpoints and took it from there. Just the keys are added for now.